skglm/experimental/tests/test_sqrt_lasso.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+from numpy.linalg import norm
+from statsmodels.regression import linear_model
```

Comment:
Their will be an error in test collection if statsmodel is not installed, and it's not obvious to the user that statsmodel is a requirement (it's not specified in setup.py -- and we don't want to put it there). 
IMO, the import must go inside the test (with a noqa), inside a try block, and the test should xfail in case of ImportError